clarity_ext/service/file_service.py

```diff
@@ -79,12 +79,12 @@ def local_shared_file(self, file_name, mode='r', extension="", modify_attached=F
 
         # Ensure that the user is only sending in a "name" (alphanumerical or spaces)
         # File paths are not allowed
-        if not re.match(r"[\w ]+", file_name):
+        if not re.match(r"[\w ]+", file_handle):
```

Comment:
This would perhaps allow a match to the partial string ie match the text between for example the slashes in a path since it doesn't anchor the beginning and end of string?

Reply:
Good catch!